scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -130,6 +153,26 @@ if [[ $AMI_TYPE == "al2keplergpu" ]]; then
 
     sudo yum install -y cuda-toolkit-11-4
     echo "exclude=*nvidia* *cuda*" | sudo tee -a /etc/yum.conf
+elif [[ $AMI_TYPE == "al2023"*"gpu" ]]; then
+    kernel_release=$(uname -r)
+
+    sudo dnf install -y kernel-devel-"$kernel_release" \
+        kernel-modules-extra-"$kernel_release" \
+        nvidia-driver \
+        nvidia-fabric-manager \
+        pciutils \
+        xorg-x11-server-Xorg \
+        oci-add-hooks \
+        libnvidia-container1 \
+        libnvidia-container-tools \
+        nvidia-container-toolkit
+
+    sudo dnf install -y cuda-drivers \
+        cuda
+
+    # https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html#configuring-docker
+    sudo nvidia-ctk runtime configure --runtime=docker
```

Comment:
we shouldn't be using docker here, this should be containerd

Reply:
For AL2023 on ECS we setup docker:
https://github.com/aws/amazon-ecs-ami/blob/57db75f52a2c4bff434e36653d5550abc4f8d492/al2023.pkr.hcl#L97-L105